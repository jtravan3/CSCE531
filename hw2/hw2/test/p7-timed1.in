x = 1
(#x+#x+x)%7919
(#(x+1)+#(x+1)+x)%7919
(#(x+2)+#(x+2)+x)%7919
(#(x+3)+#(x+3)+x)%7919
(#(x+4)+#(x+4)+x)%7919
(#(x+5)+#(x+5)+x)%7919
(#(x+6)+#(x+6)+x)%7919
(#(x+7)+#(x+7)+x)%7919
(#(x+8)+#(x+8)+x)%7919
(#(x+9)+#(x+9)+x)%7919
(#(x+10)+#(x+10)+x)%7919
(#(x+11)+#(x+11)+x)%7919
(#(x+12)+#(x+12)+x)%7919
(#(x+13)+#(x+13)+x)%7919
(#(x+14)+#(x+14)+x)%7919
(#(x+15)+#(x+15)+x)%7919
(#(x+16)+#(x+16)+x)%7919
(#(x+17)+#(x+17)+x)%7919
(#(x+18)+#(x+18)+x)%7919
(#(x+19)+#(x+19)+x)%7919
(#(x+20)+#(x+20)+x)%7919
(#(x+21)+#(x+21)+x)%7919
(#(x+22)+#(x+22)+x)%7919
(#(x+23)+#(x+23)+x)%7919
(#(x+24)+#(x+24)+x)%7919
(#(x+25)+#(x+25)+x)%7919
(#(x+26)+#(x+26)+x)%7919
(#(x+27)+#(x+27)+x)%7919
(#(x+28)+#(x+28)+x)%7919
(#(x+29)+#(x+29)+x)%7919
(#(x+30)+#(x+30)+x)%7919
(#(x+31)+#(x+31)+x)%7919
(#(x+32)+#(x+32)+x)%7919
(#(x+33)+#(x+33)+x)%7919
(#(x+34)+#(x+34)+x)%7919
(#(x+35)+#(x+35)+x)%7919
(#(x+36)+#(x+36)+x)%7919
(#(x+37)+#(x+37)+x)%7919
(#(x+38)+#(x+38)+x)%7919
(#(x+39)+#(x+39)+x)%7919
x = 3
#2
