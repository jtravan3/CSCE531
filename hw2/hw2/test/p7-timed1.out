Welcome to the Simple Expression Evaluator.
Enter one expression per line, end with ^D

   1: 1
   2: 3
   3: 7
   4: 15
   5: 31
   6: 63
   7: 127
   8: 255
   9: 511
  10: 1023
  11: 2047
  12: 4095
  13: 272
  14: 545
  15: 1091
  16: 2183
  17: 4367
  18: 816
  19: 1633
  20: 3267
  21: 6535
  22: 5152
  23: 2386
  24: 4773
  25: 1628
  26: 3257
  27: 6515
  28: 5112
  29: 2306
  30: 4613
  31: 1308
  32: 2617
  33: 5235
  34: 2552
  35: 5105
  36: 2292
  37: 4585
  38: 1252
  39: 2505
  40: 5011
  41: 2104
  42: 3
  43: 6312
  44: 
